### PR TITLE
Improve coverage of other files

### PR DESCRIPTION
**Pull request by giordano**

### CODE REVIEW

File: test/machines.jl [hunk 1]
```diff
@@ -1,10 +1,10 @@
 module TestMachines
 
-# using Revise
```
Comment (ablaom) [inline test/machines.jl]: Please don't remove them. I uncomment during testing.

File: test/machines.jl [hunk 2]
```diff
@@ -27,6 +27,12 @@ rms(yhat, y) ≈ std(y)*sqrt(1 - 1/n)
 # test an unsupervised univariate case:
 mach = machine(UnivariateStandardizer(), float.(1:5))
 @test_logs (:info, r"Training") fit!(mach)
+@test isempty(params(mach))
+
+# test a frozen NodalMachine
+stand = machine(Standardizer(), source(DataFrame(x1=rand(10))))
```
Comment (ablaom) [inline test/machines.jl]: I'd like to work towards removing DataFrames as a dependency. You can instead use `source((x1=rand(10),))` here.

**Comment by giordano** [comment]
Done

File: test/machines.jl [hunk 3]
```diff
@@ -5,6 +5,7 @@ using MLJ
 import MLJBase
 using Test
 using Statistics
+using DataFrames
```
Comment (ablaom) [inline test/machines.jl]: Please remove `using DataFrames`

Comment (giordano) [reply test/machines.jl]: Done!

**Comment by ablaom** [comment]
Many thanks

**Pull request merged**